test/unit/all.js

```diff
@@ -33,30 +33,21 @@ global.window = {
 	}
 };
 
-requirejs(
-	['test/unit/setup'],
-	function () {
-		glob(
-			'out/amd/basic-languages/*/*.test.js',
-			{ cwd: path.join(__dirname, '../../') },
-			function (err, files) {
-				if (err) {
-					console.log(err);
-					return;
-				}
-				requirejs(
-					files.map((f) => f.replace(/^out\/amd/, 'vs').replace(/\.js$/, '')),
-					function () {
-						run(); // We can launch the tests!
-					},
-					function (err) {
-						console.log(err);
-					}
-				);
+requirejs(['test/unit/setup'], function () {
+	glob(
+		'out/amd/basic-languages/*/*.test.js',
+		{ cwd: path.join(__dirname, '../../') },
+		function (err, files) {
+			if (err) {
+				console.log(err);
+				return;
 			}
-		);
-	},
-	function (err) {
-		console.log(err);
-	}
-);
+			requirejs(
+				files.map((f) => f.replace(/^out\/amd/, 'vs').replace(/\.js$/, '')),
+				function () {
+					run(); // We can launch the tests!
+				}
+			);
+		}
+	);
+});
```

Comment:
Most changes are related to Prettier. :see_no_evil: 

I only removed the error handlers.